style-guide/event.html

```diff
@@ -0,0 +1,232 @@
+	<a class="screen-reader-shortcut" href="#wsuwp-main">Skip to main content</a> <a class="screen-reader-shortcut" href="#spine-sitenav">Skip to navigation</a>
+	<div class="style-skeletal colors-default spacing-default" id="jacket">
+		<div class="fluid folio max-1386" id="binder">
+			<header class="site-header">
+				<div class="site-title">
+					WSU Events
+				</div>
+			</header>
+			<main id="wsuwp-main">
+				<article class="post type-post status-publish hentry card-single card--event">
+					<header class="card-header">
+						<p class="card-event-type">Seminar</p>
+						<h1 class="card-title">WSU Tri-Cities celebrates grand opening of student union building</h1>
+					</header>
+					<section class="card-details row halves">
+            <div class="column one">
+  						<time class="card-event-datetime" datetime="2017-11-6T19:30"><span class="card-date">Monday, November 6, 2017 - <a class="card-calendar-add" href="#">add to calendar</a></span> <span class="card-time">7:30 pm</span></time>
+  						<div class="card-location">
+  							<p class="card-address">The Torch Club - 18 Waverly Place, New York, NY 10003</p>
+  							<p class="card-location-notes">Go there and then you need to take a slight left up the stairs.</p>
+  							<p><a class="card-map-link" href="#">View directions</a></p>
+  						</div>
+  						<p class="card-cost">$12.00</p>
+  						<p><a class="card-cta button--default" href="#">Register now</a></p>
+            </div>
+            <div class="column two">
+              <figure class="card-featured-image">
+    						<a href="https://s3.wp.wsu.edu/uploads/sites/1753/2017/11/Josephs-Grainery-alt-1188x792.jpg"><img alt="Bitter Rivalry Ale uses only Washington-produced ingredients. (Photo courtesy of Joseph’s Grainery)" class="attachment-spine-medium_size size-spine-medium_size wp-post-image" height="528" sizes="(max-width: 792px) 100vw, 792px" src="https://s3.wp.wsu.edu/uploads/sites/1753/2017/11/Josephs-Grainery-alt-792x528.jpg" srcset="https://s3.wp.wsu.edu/uploads/sites/1753/2017/11/Josephs-Grainery-alt-792x528.jpg 792w, https://s3.wp.wsu.edu/uploads/sites/1753/2017/11/Josephs-Grainery-alt-396x264.jpg 396w, https://s3.wp.wsu.edu/uploads/sites/1753/2017/11/Josephs-Grainery-alt-768x512.jpg 768w, https://s3.wp.wsu.edu/uploads/sites/1753/2017/11/Josephs-Grainery-alt-990x660.jpg 990w, https://s3.wp.wsu.edu/uploads/sites/1753/2017/11/Josephs-Grainery-alt-1188x792.jpg 1188w, https://s3.wp.wsu.edu/uploads/sites/1753/2017/11/Josephs-Grainery-alt.jpg 1200w" width="792"></a>
+    						<figcaption class="wp-caption-text">
+    							Bitter Rivalry Ale uses only Washington-produced ingredients. (Photo courtesy of Joseph’s Grainery)
+    						</figcaption>
+    					</figure>
+            </div>
+            <section class="row side-right">
+              <div class="column one">
```

Comment:
Should this section or div be something like `card-content`, to match with `card-details` in the previous section?

Reply:
I just pushed up using `card-about`. ?